.expeditor/config.yml

```diff
@@ -0,0 +1,42 @@
+# Documentation available at http://expeditor-docs.es.chef.io/
+
+# Slack channel in Chef Software slack to send notifications about build failures, etc
+slack:
+  notify_channel: sustaining-notify
+
+rubygems:
+  - win32-certstore
+
+github:
+  # The tag format to use (e.g. v1.0.0)
+  version_tag_format: "v{{version}}"
+  # The Github Team primarily responsible for handling incoming Pull Requests.
+  maintainer_group: chef/msys-developers
+  # Which Github branches to build gems from, and what versions
+  # (as determined by the value in the VERSION file) those branches are responsible
+  # for building.
+  release_branch:
+    - master:
+        version_constraint: 1.*
```

Comment:
http://expeditor.es.chef.io/configuration/github/#release_branch

we should be able to drop this release-branch section. right now, this wouldn't work because we're at 0.1.0, so this setting of 1.* would prevent us from releasing.